mlflow/server/js/src/experiment-tracking/components/ExperimentListView.tsx

```diff
@@ -1,349 +1,264 @@
-          </Typography.Title>
-          <div>
-            <Tooltip componentId="mlflow.experiment_list_view.new_experiment_button.tooltip" content="New experiment">
+  return (
+    <ScrollablePageWrapper css={{ overflow: 'hidden', display: 'flex', flexDirection: 'column' }}>
+      <Spacer shrinks={false} />
+      <Header
+        title={<FormattedMessage defaultMessage="Experiments" description="Header title for the experiments page" />}
+        buttons={
+          <>
+            <Button
+              componentId="mlflow.experiment_list_view.new_experiment_button"
+              type="primary"
+              onClick={handleCreateExperiment}
+              data-testid="create-experiment-button"
+            >
+              <FormattedMessage
+                defaultMessage="Create experiment"
+                description="Label for the create experiment action on the experiments list page"
+              />
+            </Button>
+            <Tooltip
+              componentId="mlflow.experiment_list_view.compare_experiments_button"
+              content="Select at least two experiments from the table to compare them"
+            >
               <Button
                 componentId="mlflow.experiment_list_view.new_experiment_button"
-                icon={<PlusCircleIcon />}
-                onClick={this.handleCreateExperiment}
-                title="New Experiment"
+                onClick={pushExperimentRoute}
                 data-testid="create-experiment-button"
-              />
-            </Tooltip>
-            <Tooltip componentId="mlflow.experiment_list_view.hide_button.tooltip" content="Hide experiment list">
-              <Button
-                componentId="mlflow.experiment_list_view.hide_button"
-                icon={<CaretDownSquareIcon />}
-                onClick={this.hide}
-                css={{ svg: { transform: 'rotate(90deg)' } }}
-                title="Hide experiment list"
-              />
+                disabled={checkedKeys.length < 2}
+              >
+                <FormattedMessage
+                  defaultMessage="Compare experiments"
+                  description="Label for the compare experiments action on the experiments list page"
+                />
+              </Button>
             </Tooltip>
-          </div>
-        </div>
-        <Input
-          componentId="mlflow.experiment_list_view.search_input"
-          placeholder="Search experiments"
-          aria-label="search experiments"
-          value={searchInput}
-          onChange={this.handleSearchInputChange}
-          data-testid="search-experiment-input"
+          </>
+        }
+      />
+      <Spacer shrinks={false} />
+      {error && (
+        <>
+          <Alert
+            type="error"
+            message={error.message || 'A network error occurred.'}
+            componentId="mlflow.experiment_list_view.error"
+            closable={false}
+          />
+          <Spacer />
+        </>
```

Comment:
nit: does the alert component accept styling? if so we can get rid of the fragment + spacer to make this a bit more concise
```suggestion
          <Alert
            css={{ marginBottom: theme.spacing.sm }}
            type="error"
            message={error.message || 'A network error occurred.'}
            componentId="mlflow.experiment_list_view.error"
            closable={false}
          />
```